_data/simple-icons.json

```
@@ -9956,6 +9956,12 @@
             "hex": "3F6184",
             "source": "https://github.com/wallabag/logo/blob/f670395da2d85c3bbcb8dcfa8d2a339d8af5abb0/_default/icon/svg/logo-icon-black-no-bg.svg"
         },
+        {
+            "title": "Walmart",
+            "hex": "0071CE",
+            "source": "https://cdn.corporate.walmart.com/12/9d/06ecf3584e91acccc37162f09714/logowhite.svg",
+            "guidelines": "https://one.walmart.com/content/people-experience/associate-brand-center.html.html"
```

Not sure where the double `.html` comes from but it seems to work with one as well

```suggestion
            "guidelines": "https://one.walmart.com/content/people-experience/associate-brand-center.html"
```